docker-compose.yml

```diff
@@ -182,6 +182,10 @@ services:
       dockerfile: Dockerfile
     environment:
       - OPENSEARCH_ENDPOINT=http://opensearch-single-node:9200
+      - OPENSEARCH_AUTH_TYPE=none
+      - AWS_REGION=
```

Comment:
Why did you left empty `AWS_REGION` variable?

Reply:
Because there is no need to specify the region since OPENSEARCH_AUTH_TYPE is set to "none." An empty value works fine in this case.

Reply:
See how the auth initialization works when the auth_type is set to "none." It doesn't start anything.

![Screenshot 2024-06-19 at 17 51 56](https://github.com/opensupplyhub/open-supply-hub/assets/58369841/49ca3393-4be4-4e27-b520-a78994c42c06)

Reply:
Ack.